README.md

```diff
@@ -41,6 +41,8 @@ Available options:
 | replace | Replace each character with the specified string when `silent` is true | string | '' |
 | input | Input stream to read from | [Stream](https://nodejs.org/api/process.html#process_process_stdin) | process.stdin |
 | output | Output stream to write to | [Stream](https://nodejs.org/api/process.html#process_process_stdout) | process.stdout |
+| timeout | Timeout in ms | number | 0 |
+| timeoutToDefault | Return default value if timed out | boolean | false |
```

Comment:
Can you update this line to use the new name?

Reply:
Sorry i missed it